Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [monerujo/Main strings](https://hosted.weblate.org/projects/monerujo/app/app-main/).


It also includes following components:

* [monerujo/Help page](https://hosted.weblate.org/projects/monerujo/app/app-help/)

* [monerujo/About page](https://hosted.weblate.org/projects/monerujo/app/app-about/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/monerujo/app/app-main/horizontal-auto.svg)

weblate committed:
Update translation files

Updated by "Remove blank strings" add-on in Weblate.

Update translation files

Updated by "Remove blank strings" add-on in Weblate.

Update translation files

Updated by "Remove blank strings" add-on in Weblate.

Translated using Weblate (Czech)

Currently translated at 80.0% (4 of 5 strings)

Translated using Weblate (Czech)

Currently translated at 9.6% (36 of 373 strings)

Translated using Weblate (Czech)

Currently translated at 5.0% (1 of 20 strings)

Translated using Weblate (Serbian)

Currently translated at 5.0% (1 of 20 strings)

Translated using Weblate (Estonian)

Currently translated at 5.0% (1 of 20 strings)

Translated using Weblate (Serbian)

Currently translated at 0.0% (0 of 5 strings)

Translated using Weblate (Greek)

Currently translated at 0.0% (0 of 5 strings)

Translated using Weblate (Turkish)

Currently translated at 99.4% (371 of 373 strings)

Translated using Weblate (Turkish)

Currently translated at 100.0% (20 of 20 strings)

Translated using Weblate (Turkish)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Arabic)

Currently translated at 5.0% (1 of 20 strings)

Translated using Weblate (Arabic)

Currently translated at 0.0% (0 of 5 strings)

Translated using Weblate (Arabic)

Currently translated at 94.9% (354 of 373 strings)

Translated using Weblate (Arabic)

Currently translated at 94.9% (354 of 373 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 60.8% (227 of 373 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 76.4% (285 of 373 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 84.1% (314 of 373 strings)

Translated using Weblate (Slovak)

Currently translated at 77.2% (288 of 373 strings)

Translated using Weblate (Serbian)

Currently translated at 77.2% (288 of 373 strings)

Translated using Weblate (Persian)

Currently translated at 90.0% (336 of 373 strings)

Translated using Weblate (Swedish)

Currently translated at 90.6% (338 of 373 strings)

Translated using Weblate (French)

Currently translated at 99.4% (371 of 373 strings)

Translated using Weblate (French)

Currently translated at 99.4% (371 of 373 strings)

Translated using Weblate (Spanish)

Currently translated at 99.4% (371 of 373 strings)

Translated using Weblate (Dutch)

Currently translated at 68.0% (254 of 373 strings)

Translated using Weblate (Italian)

Currently translated at 88.4% (330 of 373 strings)

Translated using Weblate (Hebrew)

Currently translated at 91.6% (342 of 373 strings)

Translated using Weblate (Hungarian)

Currently translated at 65.6% (245 of 373 strings)

Translated using Weblate (Portuguese)

Currently translated at 76.6% (286 of 373 strings)

Translated using Weblate (Portuguese)

Currently translated at 76.6% (286 of 373 strings)

Translated using Weblate (Catalan)

Currently translated at 76.9% (287 of 373 strings)

Translated using Weblate (Japanese)

Currently translated at 76.6% (286 of 373 strings)

Translated using Weblate (Ukrainian)

Currently translated at 99.4% (371 of 373 strings)

Translated using Weblate (Ukrainian)

Currently translated at 99.4% (371 of 373 strings)

Translated using Weblate (Romanian)

Currently translated at 86.5% (323 of 373 strings)

Translated using Weblate (Russian)

Currently translated at 99.4% (371 of 373 strings)

Translated using Weblate (Estonian)

Currently translated at 75.0% (280 of 373 strings)

Translated using Weblate (Esperanto)

Currently translated at 75.3% (281 of 373 strings)

Translated using Weblate (Esperanto)

Currently translated at 75.3% (281 of 373 strings)

Translated using Weblate (Greek)

Currently translated at 56.8% (212 of 373 strings)

Translated using Weblate (German)

Currently translated at 96.7% (361 of 373 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 99.7% (372 of 373 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 99.7% (372 of 373 strings)

Translated using Weblate (Tamil)

Currently translated at 99.4% (371 of 373 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 45.0% (9 of 20 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 75.0% (15 of 20 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 75.0% (15 of 20 strings)

Translated using Weblate (Slovak)

Currently translated at 75.0% (15 of 20 strings)

Translated using Weblate (Persian)

Currently translated at 100.0% (20 of 20 strings)

Translated using Weblate (Swedish)

Currently translated at 45.0% (9 of 20 strings)

Translated using Weblate (French)

Currently translated at 100.0% (20 of 20 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (20 of 20 strings)

Translated using Weblate (Dutch)

Currently translated at 50.0% (10 of 20 strings)

Translated using Weblate (Italian)

Currently translated at 70.0% (14 of 20 strings)

Translated using Weblate (Hebrew)

Currently translated at 100.0% (20 of 20 strings)

Translated using Weblate (Hungarian)

Currently translated at 60.0% (12 of 20 strings)

Translated using Weblate (Portuguese)

Currently translated at 75.0% (15 of 20 strings)

Translated using Weblate (Catalan)

Currently translated at 75.0% (15 of 20 strings)

Translated using Weblate (Japanese)

Currently translated at 75.0% (15 of 20 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (20 of 20 strings)

Translated using Weblate (Romanian)

Currently translated at 75.0% (15 of 20 strings)

Translated using Weblate (Russian)

Currently translated at 95.0% (19 of 20 strings)

Translated using Weblate (Esperanto)

Currently translated at 65.0% (13 of 20 strings)

Translated using Weblate (Greek)

Currently translated at 50.0% (10 of 20 strings)

Translated using Weblate (German)

Currently translated at 80.0% (16 of 20 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (20 of 20 strings)

Translated using Weblate (Tamil)

Currently translated at 70.0% (14 of 20 strings)

Translated using Weblate (Tamil)

Currently translated at 70.0% (14 of 20 strings)

Translated using Weblate (Norwegian Bokmål)

Currently translated at 60.0% (3 of 5 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 80.0% (4 of 5 strings)

Translated using Weblate (Slovak)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Persian)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Swedish)

Currently translated at 80.0% (4 of 5 strings)

Translated using Weblate (French)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Dutch)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Hebrew)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Hungarian)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Catalan)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Japanese)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Romanian)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Russian)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Estonian)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Esperanto)

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (German)

Currently translated at 80.0% (4 of 5 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (5 of 5 strings)

Co-authored-by: Anonymous <noreply@weblate.org>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: John Doe <thraex@numericable.fr>
Co-authored-by: LucasMZ <git@lucasmz.dev>
Co-authored-by: Максим Горпиніч <maksimgorpinic4@gmail.com>
Co-authored-by: தமிழ்நேரம் <anishprabu.t@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/ar/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/ca/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/cs/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/de/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/el/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/eo/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/es/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/et/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/fa/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/fr/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/he/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/hu/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/it/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/ja/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/nl/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/pt/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/ro/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/ru/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/sk/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/sr/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/sv/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/ta/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/tr/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/uk/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ar/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ca/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/cs/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/de/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/el/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/eo/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/es/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/et/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/fa/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/fr/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/he/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/hu/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/it/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ja/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/nl/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/pt/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ro/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ru/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/sk/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/sr/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/sv/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ta/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/tr/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/uk/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/ar/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/ca/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/cs/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/de/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/el/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/eo/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/es/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/et/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/fa/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/fr/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/he/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/hu/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/it/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/ja/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/nb_NO/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/nl/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pt/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/ro/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/ru/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/sk/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/sr/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/sv/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/ta/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/tr/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/uk/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/zh_Hant/
Translation: monerujo/About page
Translation: monerujo/Help page
Translation: monerujo/Main strings

lucasmz-dev commented:
Weblate likes to mess with files when it isn't in the same shape it usually parses them, so did a force write there, then squashed 93 spammy commits into one. This is the "update translation files" commit. It pretty much just messes with lines and such, but it should mean cleaner commits from now on.

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pt_BR/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/uk/

cardpuncher committed:
Translated using Weblate (French)

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/fr/

cardpuncher committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/tr/

Commit:
Added translation using Weblate (Hindi)

Commit:
Added translation using Weblate (Hindi)

Commit:
Added translation using Weblate (Hindi)

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/uk/

cardpuncher committed:
Translated using Weblate (French)

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/fr/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (20 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/pt_BR/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/ru/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (20 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ru/

Commit:
Added translation using Weblate (Polish)

Commit:
Added translation using Weblate (Polish)

Commit:
Added translation using Weblate (Polish)

Commit:
Translated using Weblate (Polish)

Currently translated at 30.0% (6 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 80.0% (4 of 5 strings)

Translation: monerujo/About page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 30.0% (6 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 30.0% (6 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 60.5% (226 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 60.5% (226 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 60.5% (226 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 60.5% (226 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 60.5% (226 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 60.5% (226 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 60.5% (226 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 60.5% (226 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pl/

Commit:
Translated using Weblate (Polish)

Currently translated at 60.5% (226 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pl/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (5 of 5 strings)

Translation: monerujo/About page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/uk/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (5 of 5 strings)

Translation: monerujo/About page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/uk/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/uk/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (20 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/pt_BR/

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pt_BR/

Commit:
Translated using Weblate (Tamil)

Currently translated at 70.0% (14 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ta/

SantosSi committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pt/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (20 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ta/

Commit:
Translated using Weblate (Italian)

Currently translated at 87.9% (328 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/it/

Commit:
Translated using Weblate (Italian)

Currently translated at 85.0% (17 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/it/

Commit:
Translated using Weblate (Italian)

Currently translated at 95.1% (355 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/it/

TamilNeram committed:
Translated using Weblate (Tamil)

Currently translated at 100.0% (20 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ta/

TamilNeram committed:
Translated using Weblate (Tamil)

Currently translated at 100.0% (20 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ta/

TamilNeram committed:
Translated using Weblate (Tamil)

Currently translated at 100.0% (20 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ta/

TamilNeram committed:
Translated using Weblate (Tamil)

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/ta/

TamilNeram committed:
Translated using Weblate (Tamil)

Currently translated at 100.0% (5 of 5 strings)

Translation: monerujo/About page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/ta/

Commit:
Translated using Weblate (Arabic)

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/ar/

Commit:
Translated using Weblate (Hindi)

Currently translated at 9.9% (37 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/hi/

Commit:
Translated using Weblate (Hindi)

Currently translated at 60.0% (3 of 5 strings)

Translation: monerujo/About page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/hi/

xorchi committed:
Added translation using Weblate (Indonesian)

xorchi committed:
Added translation using Weblate (Indonesian)

xorchi committed:
Added translation using Weblate (Indonesian)

xorchi committed:
Translated using Weblate (Indonesian)

Currently translated at 16.8% (63 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/id/

xorchi committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (5 of 5 strings)

Translation: monerujo/About page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/id/

xorchi committed:
Translated using Weblate (Indonesian)

Currently translated at 40.0% (8 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/id/

xorchi committed:
Translated using Weblate (Indonesian)

Currently translated at 59.2% (221 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/id/

xorchi committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (5 of 5 strings)

Translation: monerujo/About page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-about/id/

xorchi committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (20 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/id/

xorchi committed:
Translated using Weblate (Indonesian)

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/id/

TamilNeram committed:
Translated using Weblate (Tamil)

Currently translated at 100.0% (20 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/ta/

Commit:
Translated using Weblate (Catalan)

Currently translated at 76.6% (286 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/ca/

SantosSi committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (20 of 20 strings)

Translation: monerujo/Help page
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-help/pt/

fgf166074-source committed:
Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (373 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/zh_Hans/

Commit:
Translated using Weblate (Polish)

Currently translated at 83.3% (311 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/pl/

Commit:
Translated using Weblate (Czech)

Currently translated at 46.3% (173 of 373 strings)

Translation: monerujo/Main strings
Translate-URL: https://hosted.weblate.org/projects/monerujo/app/app-main/cs/
